config/locales/spotlight.en.yml

```diff
@@ -109,6 +115,15 @@ en:
       fields:
         facet:
           exhibit_tag: Exhibit Tags
+  locales:
```

Comment:
I'm uncomfortable with managing the language configuration in a yaml file; maybe this could move to application or engine configuration?